bitcoin/src/transaction.rs

```diff
@@ -245,6 +253,15 @@ impl<N: BitcoinNetwork> Outpoint<N> {
                             false => None,
                         },
                     },
+                    BitcoinFormat::NATIVE_P2WSH => match redeem_script {
+                        Some(redeem_script) => match script_pub_key[0] != 0x00 as u8
+                            && script_pub_key[1] != 0x20 as u8 && script_pub_key.len() != 34 // zero [32-byte sha256(witness script)]
+                        {
+                            true => return Err(TransactionError::InvalidScriptPubKey("NATIVE_P2WSH".into())),
```

Comment:
Nit: Friendly reminder to change instances of text containing `"NATIVE_P2WSH"` to `"P2WSH"` as well when making the variable name change.